feat: OPTIC-3: Enable users to submit and pause their lead time by returning to the dm list

robot-ci-heartex opened the pull request:
Hi @Travis1282!

This PR was created in response to a PRs in upstream repo:
- https://github.com/HumanSignal/dm2/pull/225
- https://github.com/HumanSignal/label-studio-frontend/pull/1541

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5960026479

Travis1282 committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5960035202

Travis1282 committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5969070812

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-3/submit-and-exit'

Travis1282 committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5969071801

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/5969155436

Travis1282 committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6002876108

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-3/submit-and-exit'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6002876108

Travis1282 committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6020299517

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-3/submit-and-exit'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6020299517

Travis1282 committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6050261203

Travis1282 committed:
docs: LSF Update

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6050261203

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-3/submit-and-exit'

Travis1282 committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6086657187

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-3/submit-and-exit'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6086657187

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6103155356

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-3/submit-and-exit'

bmartel commented on docs/source/tags/taxonomy.md:
```diff
@@ -23,7 +23,7 @@ Use with the following data types: audio, image, HTML, paragraphs, text, time se
 | [apiUrl] | <code>string</code> |  | URL to fetch taxonomy from remote source; API should accept optional array `path` param: `apiUrl?path[]=root&path[]=child1` to return only nested children of `child1` node[^2] |
 | [leafsOnly] | <code>boolean</code> | <code>false</code> | Allow annotators to select only leaf nodes of taxonomy |
 | [showFullPath] | <code>boolean</code> | <code>false</code> | Whether to show the full path of selected items |
-| [pathSeparator] | <code>string</code> | <code>&quot;/&quot;</code> | Separator to show in the full path (default is " / ") |
```
How come this changed?

hlomzik replied:
this happens when corresponding LS PR for another ticket was not merged for some reason and those changes from LSF PR are spilled to all the further PRs until some of them got merged in

Travis1282 committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6111187504

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-3/submit-and-exit'

Travis1282 committed:
[submodules] Build static HumanSignal/label-studio-frontend

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6151446305

Travis1282 committed:
docs: LSF Update

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6151446305

Travis1282 committed:
Merge branch 'origin/develop' into 'fb-optic-3/submit-and-exit'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6151446305

robot-ci-heartex commented:
To enable Auto Merge for this PR also merge those PRs:
- https://github.com/HumanSignal/dm2/pull/225

Travis1282 committed:
[submodules] Build static HumanSignal/dm2

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/6151529499

hlomzik commented:
related changes are already merged long time ago
